cmd/runtimetest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/mount"
```

Comment:
`docker/pkg/mount` supports different platform that is why I pull this dependency.
If we just need to support Linux now, it could be a single function.